projects/ion/src/lib/tab-group/tab-group.component.ts

```diff
@@ -0,0 +1,99 @@
+import { CommonModule } from '@angular/common';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  OnInit,
+  Output,
+  SimpleChanges,
+} from '@angular/core';
+import { IonTabComponent } from '../tab/tab.component';
+import { BorderDirectionType } from '../utils/commonTypes';
+import { IonTabGroupProps, TabInGroup } from './types';
+
+@Component({
+  standalone: true,
+  selector: 'ion-tab-group',
+  imports: [CommonModule, IonTabComponent],
+  templateUrl: './tab-group.component.html',
+  styleUrls: ['./tab-group.component.scss'],
+})
+export class IonTabGroupComponent implements OnInit, OnChanges {
+  @Input() tabs!: IonTabGroupProps['tabs'];
+  @Input() direction?: IonTabGroupProps['direction'] = 'horizontal';
+  @Input() border?: IonTabGroupProps['border'] = 'bottom';
+  @Input() size?: IonTabGroupProps['size'] = 'sm';
```

Comment:
```suggestion
  @Input() direction: IonTabGroupProps['direction'] = 'horizontal';
  @Input() border: IonTabGroupProps['border'] = 'bottom';
  @Input() size: IonTabGroupProps['size'] = 'sm';
```
You can remove the `?` when you already initialize your props with a default value.